views/base.html

```diff
@@ -10,6 +10,16 @@
   <meta name="description" content="{{title}}. Data online made easy. Share and find data, quickly and easily. Simple, solid, slick. Frictionless and fast. Built for data scientists and wranglers. Managed by Datopian and Open Knowledge International">
   <meta name="keywords" content="data,datasets,data publishing,data sharing,data hosting,data package,reference data,indicators">
   <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+  <meta name="og:title" content="DataHub">
```

Comment:
I'm not sure how these meta tags work, but why we're not using dynamic titles? E.g., on https://datahub.io/core/finance-vix, I think the title should be "VIX - CBOE Volatility Index - DataHub"